soroban-sdk/src/env.rs

```diff
@@ -210,6 +210,13 @@ impl Env {
             .unwrap()
     }
 
+    /// Returns the contract call stack as a Vec
+    /// of (contractID, functionName).
```

Comment:
Could you add an example? The example doesn't need to use two contracts I think, just one would be sufficient.

Reply:
Done